src/redux/actions/user/index.js

```diff
@@ -33,21 +33,22 @@ export const fetchUserOrganizations = username => async (dispatch) => {
 
 export const fetchsUserDictionaries = username => async (dispatch) => {
   const url = `collections/?q=${''}&limit=${0}&page=${1}&verbose=true`;
+  dispatch(isFetching(true));
   try {
     const response = await instance.get(url);
     const result = filterUserPayload(username, response.data);
     dispatch(isSuccess(result, FETCH_USER_DICTIONARY));
-    dispatch(isFetching(false));
   } catch (error) {
     notify.show('an error occurred, reload the page', 'error', 3000);
   }
 };
 
-export const fetchUserData = username => (dispatch) => {
+export const fetchUserData = username => async (dispatch) => {
```

Comment:
There is no asynchronous code you are independently executing here. You have double changed the async-await blocks. Both here and in the functions you are dispatching below. Please refactor.

Reply:
Thanks for feedback @dondrzzy. However, the functions in the scope are asynchronous, if I don't use async and await, this function will dispatch isFetching as `false` before any of the asynchronous functions gets a response and this is not good for the UI as the it will hide the AJAX loader.